bang/models.py

```diff
@@ -1005,9 +1004,16 @@ class Song(MagiModel):
     owner = models.ForeignKey(User, related_name='added_songs')
     image = models.ImageField('Album cover', upload_to=uploadItem('s'))
 
-    BAND_CHOICES = Member.BAND_CHOICES
-    i_band = models.PositiveIntegerField(_('Band'), choices=i_choices(BAND_CHOICES))
-
+    SBAND_CHOICES = (
+        'Poppin\' Party',
+        'Afterglow',
+        'Pastel*Palettes',
+        'Roselia',
+        'Hello, Happy World!',
+        'Glitter*Green'
+    )
+    i_sband = models.PositiveIntegerField(_('Band'), choices=i_choices(SBAND_CHOICES))
```

Comment:
You should just call it `i_band` (no need to call it sband) and add the extra band in songs like so:

```python
BAND_CHOICES = Member.BAND_CHOICES + ['Glitter*Green']
i_band = models.PositiveIntegerField(_('Band'), choices=i_choices(BAND_CHOICES))
```

Reply:
✅